skflow/__init__.py

```diff
@@ -117,6 +117,15 @@ def predict(self, X):
             return pred
         return pred.argmax(axis=1)
 
+    def predict_proba(self, X):
+        pred = self._session.run(self._model.predictions,
```

Comment:
I see what you mean, and would make sense. Not sure how to technically implement it though.